onos-operator/charts/config-operator/templates/deployment.yaml

```diff
@@ -44,6 +44,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: GLOBAL_IMAGE_REGISTRY
+              value: {{ include "config-operator.imageregistry" . | quote }}
```

Comment:
You can just do this:
```
{{ .Values.global.image.registry | quote }}
```
Template functions are only needed for more complicated logic

Reply:
Done